### ISSUE TITLE
Bundle Error, react-native@0.63.3, WatermelonDB

### ISSUE THREAD

**Issue by Cully-Curwen** [panz3r/react-native-flipper-databases#2]
**Describe the bug**
Getting the bellow error when trying to use this module, with the WatermelonDB usage example. Occurs on bundle.

```bash
Error: While trying to resolve module `react-native-flipper-databases` from file `<ProjectFolder>/src/stores/index.ts`,
 the package `<ProjectFolder>/node_modules/react-native-flipper-databases/package.json` was successfully found.
 However, this package itself specifies a `main` module field that could not be resolved (`<ProjectFolder>/node_modules/react-native-flipper-databases/src`.
 Indeed, none of these files exist:

  * <ProjectFolder>/node_modules/react-native-flipper-databases/src(.native|.android.js|.native.js|.js|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)
  * <ProjectFolder>/node_modules/react-native-flipper-databases/src/index(.native|.android.js|.native.js|.js|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)
    at ResolutionRequest.resolveDependency (<ProjectFolder>/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:65:15)
    at DependencyGraph.resolveDependency (<ProjectFolder>/node_modules/metro/src/node-haste/DependencyGraph.js:287:16)
    at Object.resolve (<ProjectFolder>/node_modules/metro/src/lib/transformHelpers.js:267:42)
    at <ProjectFolder>/node_modules/metro/src/DeltaBundler/traverseDependencies.js:434:31
    at Array.map (<anonymous>)
    at resolveDependencies (<ProjectFolder>/node_modules/metro/src/DeltaBundler/traverseDependencies.js:431:18)
    at <ProjectFolder>/node_modules/metro/src/DeltaBundler/traverseDependencies.js:275:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (<ProjectFolder>/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
    at _next (<ProjectFolder>/node_modules/metro/src/DeltaBundler/traverseDependencies.js:107:9)
```

**To Reproduce**
react-native: 0.63.3
using Hermes Engine

Flipper version: 0.64.0

Installed with npm on Ubuntu 20.10,
also with yarn on Windows 10 

Code from usage example used.
```ts
if (__DEV__) {
  // Import connectDatabases function
  const connectDatabases = require('react-native-flipper-databases').default;

  // Import required DBDrivers
  const WatermelonDBDriver = require('react-native-flipper-databases/drivers/watermelondb').default;

  connectDatabases([
    new WatermelonDBDriver(database), // Pass in database definition
  ]);
}
```
As well as using,
```ts
import connectDatabases from 'react-native-flipper-databases';
import WatermelonDBDriver from 'react-native-flipper-databases/drivers/watermelondb';

....

if (__DEV__) {
   connectDatabases([
    new WatermelonDBDriver(database),
  ]);
}
```

**Comment by panz3r** [comment]
Hi @Cully-Curwen,

Sorry for the late answer, the reported issue should be resolved with the latest `0.2.1` version.

**Comment by Cully-Curwen** [comment]
Hi @panz3r,

No worries, with `0.2.1` all is working on my end.
Thank you for your time and effort.
